openedx/adg/admin_dashboard/admin_settings/tests/test_views.py

```diff
@@ -188,3 +189,53 @@ def test_admin_account_list_view_and_pending_invitation_list_view_post_method(
     }
 
     mock_render.assert_called_once_with(request, f'adg/admin_dashboard/admin_settings/{template}', expected_context)
+
+
+@pytest.mark.django_db
+@skip_unless_lms
+def test_update_permissions__adg_admin(request_factory, super_user):
```

Comment:
Is double underscore intentional?

Reply:
Yes its intentional. Basically it will tell what exact scenario we are testing in this test case. For example in the above case we are testing `test_update_permissions` for adg_admin